src/eva/utilities/utils.py

```diff
@@ -15,6 +17,15 @@
 # --------------------------------------------------------------------------------------------------
 
 
+class fontColors:
```

Comment:
very nice!

Reply:
I did wonder if this might be unpredictable say on Windows. But I'm guessing there might be other problems trying to run on Windows.